<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>AboutDialog</class>
<widget class="QDialog" name="AboutDialog">
<property name="geometry">
<rect>
<x>0</x>
<y>0</y>
<width>190</width>
<height>167</height>
</rect>
</property>

<property name="sizePolicy">
<sizepolicy hsizetype="Minimum" vsizetype="Minimum">
<horstretch>0</horstretch>
<verstretch>0</verstretch>
</sizepolicy>
</property>

<property name="windowTitle">
<string>TIO browse - about</string>
</property>

<layout class="QVBoxLayout" name="vertialLayout">

<item>
<layout class="QGridLayout" name="gridLayout">

<item row="2" column="0" colspan="2">
<widget class="QLabel" name="appDescriptionLabel">
<property name="text">
<string>appDescription</string>
</property>
</widget>
</item>

<item row="0" column="0" colspan="2">
<widget class="QLabel" name="appNameLabel">
<property name="text">
<string>appName</string>
</property>
</widget>
</item>

<item row="1" column="1">
<widget class="QLabel" name="appVersionLabel">
<property name="text">
<string>appVersion</string>
</property>
</widget>
</item>

<item row="3" column="1">
<widget class="QLabel" name="custodianNameLabel">
<property name="text">
<string>custodianName</string>
</property>
</widget>
</item>

<item row="3" column="0">
<widget class="QLabel" name="custodianLabel">
<property name="text">
<string>Custodian</string>
</property>
</widget>
</item>

<item row="1" column="0">
<widget class="QLabel" name="versionLabel">
<property name="text">
<string>Version</string>
</property>
</widget>
</item>

<item row="4" column="0">
<widget class="QLabel" name="extensionLabel">
<property name="text">
<string>Extension</string>
</property>
</widget>
</item>

<item row="5" column="0">
<widget class="QLabel" name="emailLabel">
<property name="text">
<string>email</string>
</property>
</widget>
</item>

<item row="5" column="1">
<widget class="QLabel" name="custodianEmailLabel">
<property name="text">
<string>custodianEmail</string>
</property>
</widget>
</item>

<item row="4" column="1">
<widget class="QLabel" name="custodianExtensionLabel">
<property name="sizePolicy">
<sizepolicy hsizetype="Minimum" vsizetype="Minimum">
<horstretch>0</horstretch>
<verstretch>0</verstretch>
</sizepolicy>
</property>
<property name="text">
<string>custodianExtension</string>
</property>
</widget>
</item>

</layout>
</item>


<item>
<widget class="QDialogButtonBox" name="buttonBox">
<property name="orientation">
<enum>Qt::Horizontal</enum>
</property>
<property name="standardButtons">
<set>QDialogButtonBox::Ok</set>
</property>
<property name="centerButtons">
<bool>true</bool>
</property>
</widget>
</item>

</layout>

</widget>
<resources/>

<connections>

<connection>
<sender>buttonBox</sender>
<signal>accepted()</signal>
<receiver>AboutDialog</receiver>
<slot>accept()</slot>
<hints>
<hint type="sourcelabel">
<x>248</x>
<y>254</y>
</hint>
<hint type="destinationlabel">
<x>157</x>
<y>274</y>
</hint>
</hints>
</connection>

<connection>
<sender>buttonBox</sender>
<signal>rejected()</signal>
<receiver>AboutDialog</receiver>
<slot>reject()</slot>
<hints>
<hint type="sourcelabel">
<x>316</x>
<y>260</y>
</hint>
<hint type="destinationlabel">
<x>286</x>
<y>274</y>
</hint>
</hints>
</connection>

</connections>
</ui>
